ACHNBrowserUI/ACHNBrowserUI/viewModels/CategoriesSearchViewModel.swift

```diff
@@ -17,25 +17,22 @@ class CategoriesSearchViewModel: ObservableObject {
     private var searchCancellable: AnyCancellable?
     
     init() {
-        searchCancellable = _searchText
-            .projectedValue
+        searchCancellable = $searchText
             .debounce(for: .milliseconds(300), scheduler: DispatchQueue.main)
             .removeDuplicates()
             .filter { !$0.isEmpty }
             .map(itemsInCategories(with:))
-            .assign(to: \.searchResults, on: self)
+            .subscribe(on: DispatchQueue.global())
+            .receive(on: DispatchQueue.main)
```

Comment:
adding
```swift 
.subscribe(on: DispatchQueue.global())
.receive(on: DispatchQueue.main)
```
will help to avoid UI crashes.
However, it's still a bit laggy let's say. But it's not related to which thread is used (as you explained me the other day ;))